graal/com.oracle.graal.graph/src/com/oracle/graal/graph/NodeBitMap.java

```
@@ -32,13 +32,13 @@
 
     private long[] bits;
     private int nodeCount;
-    private final NodeIdAccessor nodeIdAccessor;
```

A quick scan of `NodeBitMap` uses seems to indicate that it could implement `AutoCloseable` and be used in try-with-resource statements where closing the map causes further operations to fail fast. In addition, if `NodeBitMap`s are only ever obtained from `Graph.createNodeBitMap()`, then one or two instances could be cached for reuse in a graph.